### DISCUSSION TITLE
Mastodon

### DISCUSSION THREAD

**Discussion by dgmltn** [johnlindquist/kit#1343] [Ideas]
I'd like to share my Mastodon account instead of X/Formerly-known-as-Twitter. Is there a way to show that in a script's metadata? It'd be cool if it were a first class citizen of metadata `// Mastodon: me@server.com`, or if there were another more generic `// Social: url` type of metadata.

**Comment by johnlindquist** [comment]
@dgmltn You can use this metadata as a generic link:

```js
// Social: John Lindquist
// Social_Url: https://johnlindquist.com
```

**Reply by dgmltn** [reply]
I should've known it's already addressed. Thanks! Sure enough, I found those metadata [here](https://github.com/johnlindquist/kit/blob/eee929f25dca4049107de212d56cb4d7ba3cc53b/src/types/core.d.ts#L98).

My feature request is to update [the guide](https://github.com/johnlindquist/kit/discussions/185), then.
